files/en-us/web/html/element/input/index.html

```diff
@@ -397,7 +397,7 @@ <h2 id="Attributes">Attributes</h2>
   <tr>
    <td>{{anch('htmlattrdefvalue', 'value')}}</td>
    <td>all</td>
-   <td>At first, the initial value if specified explicitly in HTML. More generally, the current value of the form control. Submitted with the form as part of a name/value pair.</td>
+   <td>The default value. Not to be confuse with the <a href="https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement">HTMLInputElement.value</a> property - which gets/sets the <code>&lt;input&gt;</code>'s current value.</td>
```

Comment:
```suggestion
   <td>The initial value of the control.</td>
```

I think "initial" is better than "default" here, and I don't think we need to refer to `HTMLInputElement` at all here: the same is true of all the attributes that `HTMLInputElement` reflects.

Reply:
Thanks for the feedback!
I agree.

After further (but not exhaustive) investigation, I found the only time using the HTMLInputElement.value property to set the HTML element's value attribute does not work, is for input's whose type attribute is set to _text_, or _number_ etc.

For example:
Given the HTML
`<input id="a" type="submit" value="submit this form"/>`
the JavaScript
`document.getElementById('a').value = 'new text';`
**will** change the value attribute.

But, given
`<input id="b" type="number" value="0"/>`
the JavaScript
`document.getElementById('b').value = 4;`
will **not** change the value attribute. (but will change the value seen in the browser, and the value submitted)

Thanks for your help!